components/brave_wallet/browser/keyring_service.cc

```diff
@@ -1227,16 +1256,12 @@ std::vector<mojom::AccountInfoPtr> KeyringService::GetHardwareAccountsSync()
 
 // static
 base::Value* KeyringService::GetPrefForHardwareKeyringUpdate(
-    PrefService* prefs) {
+    PrefService* prefs,
+    const std::string& keyring_id) {
   DCHECK(prefs);
   DictionaryPrefUpdate update(prefs, kBraveWalletKeyrings);
```

Comment:
update is not used anymore, should remove it.

Reply:
done